database/seeds/RegistrationsSeeder.php

```diff
@@ -37,9 +37,120 @@ public function run()
         // create a registration with a bundle
         $bundle = factory(App\Bundle::class)->create();
         factory(App\Registration::class)->create()->bundles()->save($bundle);
-    }
+
+        // centre without SK rules for fam1 & fam4 that is not in same area as families 3, 5 and 6
```

Comment:
This won't be SK rules after this sprint, would it make sense to call it family rule? Or disqualifier rule? @charlesstrange2 thoughts?

Reply:
Where you're checking child requires verification, use "child verification rules"
Where you're checking child qualification use "child qualification rules"

Reply:
how can I tell which one applies to which test family?